subsys/bluetooth/shell/bt.c

```diff
@@ -1122,6 +1122,40 @@ static int cmd_bondable(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+static void bond_info(const struct bt_bond_info *info,void *user_data)
+{
+	int *bond_count = (int *)user_data;
+	char addr[BT_ADDR_LE_STR_LEN];
+	bool connected = false;
+	struct bt_conn * conn;
```

Comment:
minor: no space between the `*` and the variable name

Reply:
Fixed.